prereise/gather/demanddata/transportation_electrification/immediate_charging_HDV.py

```diff
@@ -50,27 +50,27 @@ def calculate_charging_helper(
         * charging_efficiency
     )
 
-    # -- setting values in the group whose Trip Number == 1 --
+    # -- setting values in the group whose trip_number == 1 --
     # they don't necessarily have to be the first in the group bc of how they were grouped
-    group.loc[group["Trip Number"] == 1, "trip start battery charge"] = battery_capacity
+    group.loc[group["trip_number"] == 1, "trip start battery charge"] = battery_capacity
```

Comment:
Looks like we aren't changing the `trip_number` column - if that's correct, consider defining e.g. `trip_no_1 = group["trip_number"] == 1` since we are using that filter a lot.